server/sonar-server/src/main/java/org/sonar/server/email/ws/SendAction.java

```diff
@@ -74,7 +73,7 @@ public void define(WebService.NewController context) {
 
   @Override
   public void handle(Request request, Response response) throws Exception {
-    userSession.checkPermission(GlobalPermissions.SYSTEM_ADMIN);
+    userSession.checkLoggedIn().checkIsRoot();
```

Comment:
either way, description should be fixed, root is now required, not system administration... or is that the name we give to root?